shell/platform/android/platform_view_android_jni.cc

```diff
@@ -174,18 +174,26 @@ std::unique_ptr<IsolateConfiguration> CreateIsolateConfiguration(
   const auto configuration_from_blob =
       [&asset_manager](const std::string& snapshot_name)
       -> std::unique_ptr<IsolateConfiguration> {
-    std::unique_ptr<fml::Mapping> blob =
-        asset_manager.GetAsMapping(snapshot_name);
+    auto blob = asset_manager.GetAsMapping(snapshot_name);
+    auto delta = asset_manager.GetAsMapping("kernel_delta.bin");
+    if (blob && delta) {
+      std::vector<std::unique_ptr<fml::Mapping>> kernels;
+      kernels.emplace_back(std::move(blob));
+      kernels.emplace_back(std::move(delta));
```

Comment:
So "blob" is the common part with the framework that would otherwise go in the core snapshot, and "delta" is the individual card without the framework?

Reply:
In this case, it will load 3 artifacts: platform.dill with the framework, kernel_blob.bin with the host app, and kernel_delta.bin with the card.